requirements.txt

```diff
@@ -14,6 +14,7 @@ django-toolbelt==0.0.1
 django-treebeard==4.0.1
 djangorestframework==3.4.6
 docutils==0.12
+elasticsearch==1.2.0
```

Comment:
Do any changes to the `README.md` need to be made to explain getting elastic working?